local/modules/Tinymce/Tinymce.php

```diff
@@ -41,6 +43,16 @@ public function postActivation(ConnectionInterface $con = null)
     {
         $fileSystem = new Filesystem();
 
+        //Check for environment
+        if ($env = $this->getContainer()->getParameter('kernel.environment')) {
+            //Check for backward compatibility
+            if ($env !== "prod") {
```

Comment:
this particular case doesn't cover default Thelia's behaviour. I can be in development mode and the folder ```media/dev``` will be created and never used. Moreover, the ```_dev``` part is never removed in the config file, so were files will be copied ?